src/transformers/models/fuyu/modeling_fuyu.py

```diff
@@ -56,7 +56,7 @@ class FuyuPreTrainedModel(PreTrainedModel):
     config_class = FuyuConfig
     base_model_prefix = "fuyu"
     supports_gradient_checkpointing = True
-    _no_split_modules = []
+    _no_split_modules = ["PersimmonDecoderLayer"]
```

Comment:
Bit bothered by this and the reason why I didn’t add it: we have an auto model and the no split module should be inferred from it. Persimmon doesn’t have to be the langage model here

Reply:
sounds good